metricbeat/module/openmetrics/collector/collector_test.go

```diff
@@ -43,23 +43,23 @@ func TestData(t *testing.T) {
 }
 
 func TestSameLabels(t *testing.T) {
-	dataConfig := mbtest.ReadDataConfig(t, "_meta/samelabeltestdata/config.yml")
-	mbtest.TestDataFilesWithConfig(t, "openmetrics", "collector", dataConfig)
+	mbtest.TestDataFiles(t, "openmetrics", "collector")
 }
+
 func TestGetOpenMetricsEventsFromMetricFamily(t *testing.T) {
 	labels := mapstr.M{
 		"handler": "query",
 	}
 	tests := []struct {
-		Family *p.MetricFamily
+		Family *prometheus.MetricFamily
 		Event  []OpenMetricEvent
 	}{
 		{
-			Family: &p.MetricFamily{
+			Family: &prometheus.MetricFamily{
```

Comment:
are those changes related to this PR?

Reply:
Yes, this needs to be changed in this PR. Otherwise tests will keep failing. I checked and one of the folder was a duplicate, so I deleted it to fix the error.

Reply:
@constanca-m I don't think it is a duplicate, module configuration is different - in `testdata` https://github.com/elastic/beats/blob/3268f335f0d07c306d9c38ab1f96dc4498a8a0c8/metricbeat/module/openmetrics/collector/_meta/testdata/config.yml#L6-L8 
compare to `samelabeltestdata` - https://github.com/elastic/beats/blob/3268f335f0d07c306d9c38ab1f96dc4498a8a0c8/metricbeat/module/openmetrics/collector/_meta/samelabeltestdata/config.yml#L6-L8

Reply:
`testdata` folder is not being used anywhere

Reply:
> testdata folder is not being used anywhere

what do you mean by this? tests were running only for `samelabeltestdata`, or ?

this extra tests folder with different module configuration was added in this PR https://github.com/elastic/beats/pull/27269. @ChrsMark maybe you remember why/if here we need to keep both tests with different configurations?

Reply:
> tests were running only for `samelabeltestdata`

Yes

Reply:
From what I can see, these settings have to do with how the data are grouped at https://github.com/elastic/beats/blob/9754d2b04f661e797cae113116785509d7d5ec89/metricbeat/module/openmetrics/collector/collector.go#L160-L164. So this is why there are 2 different testdata folders.